### PR TITLE
[1LP][RFR] Migration without conversion hosts

### PR DESCRIPTION
**Pull request by sshveta**
{{ pytest: cfme/tests/v2v/test_v2v_migrations_ui.py --use-provider rhv43 --use-provider vsphere67-ims --provider-limit 2 -vvvv --long-running -k test_migration_with_no_conversion }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 7fc3df7ca3c8585cc64e1332e51e37f6a427329d

The *global* fixture **`delete_conversion_hosts`** is used in the following files:
- **cfme/tests/v2v/test_v2v_migrations_ui.py**
    - *test_migration_with_no_conversion*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/v2v/test_v2v_migrations_ui.py [hunk 1]
```diff
@@ -519,3 +520,34 @@ def test_duplicate_mapping_name(appliance, mapping_data_vm_obj_mini):
...
+        assignee: sshveta
+        initialEstimate: 1/2h
+        caseimportance: high
+        caseposneg: negative
+        testtype: functional
+        startsin: 5.10
+        casecomponent: V2V
+    """
+    migration_plan_collection = appliance.collections.v2v_migration_plans
+    migration_plan = migration_plan_collection.create(
```
Comment (john-dupuy) [inline cfme/tests/v2v/test_v2v_migrations_ui.py]: Any cleanup of `migration_plan` necessary?

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 326053f7ca18c43ec9c00e3740b450119c6de594

The *global* fixture **`delete_conversion_hosts`** is used in the following files:
- **cfme/tests/v2v/test_v2v_migrations_ui.py**
    - *test_migration_with_no_conversion*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit a97957e7fd05104fffa8b4923844030d947a5937

The *global* fixture **`delete_conversion_hosts`** is used in the following files:
- **cfme/tests/v2v/test_v2v_migrations_ui.py**
    - *test_migration_with_no_conversion*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit dccf70dcd7d29ffc4323986afbdd63e452029c03

The *global* fixture **`delete_conversion_hosts`** is used in the following files:
- **cfme/tests/v2v/test_v2v_migrations_ui.py**
    - *test_migration_with_no_conversion*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 36489c0311f3b9d4aa83dd1ec4c1dad80c09de95

The *global* fixture **`delete_conversion_hosts`** is used in the following files:
- **cfme/tests/v2v/test_v2v_migrations_ui.py**
    - *test_migration_with_no_conversion*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/v2v/migration_plans.py [hunk 2]
```diff
@@ -468,6 +468,11 @@ def wait_for_state(self, state):
         except TimedOutError:
             logger.info("Wait for state :%s timed out", state)
 
+    def delete_completed_plan(self):
```
Comment (john-dupuy) [inline cfme/v2v/migration_plans.py]: Thought for the future, doesn't have to be included in this PR. Would it be possible to merged `delete_completed_plan` and `delete_not_started_plan` into a single `delete` method? 

As a user, I would expect that `delete` will delete the plan regardless of its status. Something like
```python
def delete(self):
    if self.completed():
        self.delete_completed_plan()
    else:
        self.delete_not_started_plan()
```

Comment (sshveta) [reply cfme/v2v/migration_plans.py]: @john-dupuy : Yeah I thought it too but the list to delete plan is different in each page so it need some time so created this method for time being . will do that in next PR :+1:

**Pull request merged**